module/solutions/helper.js

```diff
@@ -3159,7 +3120,6 @@ module.exports = class SolutionsHelper {
 					let solutionAndProgramCreation = await this.createProgramAndSolution(
 						userId,
 						programAndSolutionData,
-						userToken,
```

Comment:
@Prajwal17Tunerlabs please confirm deletion of this param won't cause issue, if this same function is being called from some other places in the code